README.md

```diff
@@ -730,6 +730,7 @@ The OpenTelemetry Operator _might_ work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes     | Cert-Manager |
 |------------------------| -------------- | ------------ |
+| v0.101.0               | v1.23 to v1.29 | v1           |
```

Comment:
i believe this should be bumped to 1.30 now that #2975 is merged?

Reply:
Updated